src/logic/operate.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from 'react-test-renderer';
+import Quotes from '../components/Quotes';
+
+it('Qoute page is working', () => {
+  const tree = renderer.create(<Quotes />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] As mentioned in the project requirements all the `functionalities` & `scenarios` of the `operate.js` module should be covered using Jest, please kindly add them 🙏🏾
- [ ] Also add Jest snapshots for testing the `Home` page, you only test the `Calculator` and Quote` pages 🙃